PokemonSearch/PokemonSearch/SearchTableViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SearchTableViewCell.swift
+//  PokemonSearch
+//
+//  Created by Jocelyn Stuart on 1/25/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
If you have an action on a custom cell and it's aim to change your model object, or data source array (pokemons) you need to create a protocol and delegate this responsibility to ViewController
```suggestion
protocol SearchTableViewCellDelegate: class {
    func saveButtonTapped(on cell: SearchTableViewCell)
}
```